1-mandatory/1-writers.js

```diff
@@ -71,8 +88,20 @@ Exercise 3:
   "Hi, my name is {firstName} {lastName}. I am {age} years old."
 */
 
+function getAliveWritersInTheirForties(writer) {
+  return writer.age >= 40 && writer.age <= 49 && writer.alive === true;
+}
+
 function logAliveWritersInTheirForties() {
-  // write your code here
+  let aliveWritersInTheirForties = writers.filter(
+    getAliveWritersInTheirForties
+  );
+
+  aliveWritersInTheirForties.map((writer) => {
```

Comment:
Issue: `map` isn't appropriate here as we're not modifying the items in the array. I think a `forEach` would be appropriate as we're taking an action on every item.